### ISSUE TITLE
chore: add `Deutsch - de-DE` language support

### ISSUE THREAD

**Issue by brionmario** [asgardeo/javascript#160]
### Current Limitation

Currently, the Asgardeo JavaScript SDK does not have support for **German (Deutsch - de-DE)**. Users who prefer German will not see UI components translated, limiting accessibility for German-speaking users.

> [!Important]
> Contributors for this task should be **native German speakers or have professional fluency** to ensure accurate, culturally appropriate translations.

### Suggested Improvement

Add **German (`de-DE`) language support** to the `@asgardeo/i18n` package.

**Implementation Plan:**

1. Create a new locale file:

    Follow https://github.com/asgardeo/javascript/blob/main/CONTRIBUTING.md#contributing-to-the-internalization-i18n-package

3. Add documentation for usage:  
   - Update the SDK docs: https://wso2.com/asgardeo/docs/sdks/react/overview/

### Please select the package issue is related to

@asgardeo/react, @asgardeo/nextjs

### Version

current

### Reporter Checklist

- [x] I have searched the existing issues and this is not a duplicate.
- [x] I have provided all the necessary information.
- [x] I have verified the improvement is not available in the latest version of the package.

**Comment by Ratheshan03** [comment]
Hi there @sagara-gunathunga 

Even though you require a native GER speaker, I myself have good fluency in German with language certification so should i give it a try ?
